.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+@neotrow
```

Comment:
not sure what i have to add here, any other names? or remove them?

Reply:
fine for me :-)